catroid/src/main/res/values/strings.xml

```diff
@@ -1270,6 +1270,11 @@
     <string name="brick_web_request">Send web request to</string>
     <string name="brick_web_request_result">and store answer in</string>
     <string name="brick_web_request_default_value">https://official-joke-api.appspot.com/random_joke</string>
+    <string name="brick_store_csv_into_userlist_store">Store column #</string>
+    <string name="brick_store_csv_into_userlist_csv">of CSV</string>
+    <string name="brick_store_csv_into_userlist_into">into list</string>
+    <string name="brick_store_csv_into_userlist_data">CSV data</string>
```

Comment:
```suggestion
    <string name="brick_store_csv_into_userlist_data">cat,cute
puppy,naughty
octopus,intelligent</string>
```

Please change it as suggested so that it is a more suggestive and understandable example for our users how to use this brick.